plugins/reference-assets/android/defs.bzl

```diff
@@ -0,0 +1,45 @@
+load("//jvm/dependencies:versions.bzl", "versions")
+load("@rules_player//kotlin:defs.bzl", "lint")
+load("@rules_kotlin//kotlin:android.bzl", "kt_android_local_test")
+load("//plugins/reference-assets/android/src/androidTest/java/com/intuit/playerui/android/reference/assets/test:defs.bzl", maven_test = "maven")
+
+maven_main = []
+
+main_exports = [
+    "//android/player",
+]
+
+main_deps = main_exports + [
+    "//jvm:kotlin_serialization",
+    "//plugins/reference-assets/jvm:reference-assets",
+    "//plugins/pending-transaction/jvm:pending-transaction",
+]
+
+maven = maven_main + maven_test
```

Comment:
Should these be in here?

Reply:
removed what isn't necessary, but if you mean should the other required ones be in here, I don't see why not